test/invariant/stores/TimestampStore.sol

```diff
@@ -5,12 +5,20 @@ pragma solidity >=0.8.19 <0.9.0;
 /// save the current timestamp in a contract with persistent storage.
 contract TimestampStore {
     uint256 public currentTimestamp;
+    uint256 public currentBlockNumber;
 
     constructor() {
         currentTimestamp = block.timestamp;
+        currentBlockNumber = block.number;
     }
 
     function increaseCurrentTimestamp(uint256 timeJump) external {
         currentTimestamp += timeJump;
+        currentBlockNumber += 1;
+    }
+
+    function increaseCurrentBlockNumber(uint256 blockJump) external {
+        currentTimestamp += 1;
+        currentBlockNumber += blockJump;
```

Comment:
I think it would be better to increase only timestamp and only blocknumber in those two functions for a better understanding when using them

Reply:
We need to do so, otherwise we are just changing the timestamp of the block and therefore transaction that has been passed previously will have a discrepancy in the timestamp/block stored